### ISSUE TITLE
Updating Inertia Packages

### ISSUE THREAD

**Issue by anubra266** [inertiajs/inertia#299]
@reinink  I'll like to know what way is best recommended to update all my inertia packages.

I upgraded the Inertia/inertia with 
`npm i @inertiajs/inertia@0.6.0`

Tried to upgrade the react version with 
`npm i @inertiajs/inertia@0.6.0`
 But getting

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: undefined@undefined
npm ERR! Found: @inertiajs/inertia@0.6.0
npm ERR! node_modules/@inertiajs/inertia
npm ERR!   @inertiajs/inertia@"^0.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @inertiajs/inertia@"^0.5.0" from @inertiajs/inertia-react@0.4.1
npm ERR! node_modules/@inertiajs/inertia-react
npm ERR!   @inertiajs/inertia-react@"0.4.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/abraham/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/abraham/.npm/_logs/2020-10-28T16_39_27_030Z-debug.log

```
Thanks.

**Comment by claudiodekker** [comment]
Hi,

After a very quick Google search, I discovered that this is an NPM error and not an Inertia issue, and it used to happen fairly recently when a peer dependency could not be resolved: https://github.com/npm/cli/issues/1681#issuecomment-697982601

Looking at the error a bit closer, this makes perfect sense: Since you are _only_ trying to install `@inertiajs/inertia@^0.6.0`, NPM looks at the existing & matching dependencies, and concludes that it cannot install it because your already-installed/matching `@inertiajs/inertia-react@^0.4.1` only supports `@inertiajs/inertia@^0.5.0`.

Long story short, you need to upgrade them at the same time, or first remove both, then install them one by one.

**Comment by anubra266** [comment]
Thanks a lot!
